synapse/handlers/room.py

```diff
@@ -316,6 +327,8 @@ def send(etype, content, **kwargs):
                 for invitee in invite_list:
                     power_level_content["users"][invitee] = 100
 
+            power_level_content.update(power_level_content_override)
```

Comment:
Ah, I see that you are using the overrides as part of the shutdown API.
The override stuff is trivial enough for me to to review as part of this PR.
It just looked orthogonal to me at first, hence me asking whether you meant to push it as part of the PR :)